clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/model/EcsInstance.java

```diff
@@ -0,0 +1,82 @@
+package com.netflix.spinnaker.clouddriver.ecs.model;
+
+import com.amazonaws.services.ec2.model.InstanceStatus;
+import com.amazonaws.services.ecs.model.Task;
+import com.netflix.spinnaker.clouddriver.ecs.EcsCloudProvider;
+import com.netflix.spinnaker.clouddriver.model.HealthState;
+import com.netflix.spinnaker.clouddriver.model.Instance;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+public class EcsInstance implements Instance, Serializable {
```

Comment:
The name "EcsInstance" is confusing because you could think it's about the EC2 instance, the host, that launches the docker images.  Perhaps EcsTask would be a clearer name?

![image](https://user-images.githubusercontent.com/10307100/29740550-0d95f324-8a27-11e7-8376-b7514e21a3ae.png)